ticket_metrics.view.lkml

```diff
@@ -624,6 +641,15 @@ view: ticket_metrics {
     sql: ${TABLE}.full_resolution_time_in_minutes__calendar/60 ;;
   }
 
+  measure: p95_coalesce_reply_time_and_first_resolution_time_in_calendar_hours {
+    description: "95th percentile of the number of hours it takes for a ticket to be responded to or first resolved"
+    group_label: "First Reply Time"
+    type: percentile
+    percentile: 95
+    value_format: "0.#"
```

Comment:
Your source dimension is using 2 decimal, and you shouldn't suddenly cut it to 1 decimal in measure. "0.##"

Reply:
This applies to the other measures as well.

Reply:
changed